### PR TITLE
Added a note in footer

### PR DESCRIPTION
**Pull request by mickaelandrieu**
First test of Website build with platform.sh

### CODE REVIEW

File: _themes/sphinx_rtd_theme/footer.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
   {%- if show_sphinx %}
   {% trans %}Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>{% endtrans %}.
   {%- endif %}
-
+  <p>Deployed with <a href="https://platform.sh" title="Platform.sh">Platform.sh</a></p>
```
Comment (nidup) [inline _themes/sphinx_rtd_theme/footer.html]: maybe a small logo ?

Comment (mickaelandrieu) [inline _themes/sphinx_rtd_theme/footer.html]: Yes, I'm waiting for assets :) for now platform.sh GitHub integration don't works :(

Comment (nidup) [inline _themes/sphinx_rtd_theme/footer.html]: ok!

**Comment by a2xchip** [comment]
is not there way to not use inline styles? @mickaelandrieu

**Comment by a2xchip** [comment]
There were status.github.com issues today... may be it is the reason

**Comment by mickaelandrieu** [comment]
No, was error from my side :)

Now it's time to rebase this PR and redesign the footer :)

**Comment by a2xchip** [comment]
:-D

**Pull request merged**